bitnami/jasperreports/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: jasperreports
-version: 7.0.14
```

Comment:
As you are adding a new feature you should bump the minor version

Reply:
should be included in bitnami/jasperreports/Chart.yaml

Reply:
@miguelaeh is it OK now?